File: modulo1/segmento3/apunte4/Ejercicio3.web/ejercicio3.js
function ejercicio3() {
    const letter = prompt('Ingrese un numero romano:');
    switch(letter) {
    case 'I':
    alert('Es : 1 ');
    break;
    case 'V':
    alert('Es : 5');
    break;
    case 'X':
    alert('Es : 10');
    break;
    case 'L':
    alert('Es : 50 ');
    break;
    case 'C':
    alert('Es : 100');
    break;
    case 'D':
    alert('Es : 500');
    break;
    case 'M':
    alert('Es : 1000')
    default:
    alert('ERROR: No es un numero romano!!!! ');
    break;
    }
    }

File: modulo1/segmento3/apunte4/Ejercicio3.java
import java.util.Scanner;

public class Ejercicio4 {
	public static void main(String args[]){
		Scanner keyboard = new Scanner(System.in);

		System.out.print("Ingrese un numero romano: ");
		String letter = keyboard.nextLine();

		switch(letter) {
			case "I":
				System.out.println("1");
				break;

			case "V":
				System.out.println("5");
				break;

			case "X":
				System.out.println("10");
				break;
			case "L":
				System.out.println("50");
				break;
			case "C":
				System.out.println("100");
				break;
			case "D":
				System.out.println("500");
				break;
			case "M":
				System.out.println("1000");
				break;
			default:
				System.out.println("Error!");
				break;
		}
	}
}



